src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -10,21 +14,23 @@ export function WhaleSightingViewer() {
 			<ul className="whale-sighting-posts">
 				<li className="whale-sighting-post">
 					Post1 goes here
-					<button>Like</button>
+					{props.loggedIn ? <button>Like</button> : <></>}
 				</li>
 				<li className="whale-sighting-post">
 					Post2 goes here
-					<button>Like</button>
+					{props.loggedIn ? <button>Like</button> : <></>}
 				</li>
 				<li className="whale-sighting-post">
 					Post3 goes here
-					<button>Like</button>
+					{props.loggedIn ? <button>Like</button> : <></>}
 				</li>
 				<li className="whale-sighting-post">
 					Post4 goes here
-					<button>Like</button>
+					{props.loggedIn ? <button>Like</button> : <></>}
 				</li>
 			</ul>
+			{!props.loggedIn ? <h3>Log in below to like posts:</h3> : <></>}
+			{!props.loggedIn ? <Login /> : <></>}
```

Comment:
We should be able to do this is one statement
```suggestion
			{!props.loggedIn 
			    ? <h3>Log in below to like posts:</h3>
			        <Login /> 
			    : <></>}
```